tests/validation.test.js

```
@@ -0,0 +1,370 @@
+      });
+      
+      const score = validator.validatePromptQuality(lowQualityContent, 'test.md', 'command');
+      expect(score).toBeLessThan(50);
+    });
+  });
+
+  describe('Performance Validation', () => {
+    const shouldSkipPerfTests = process.env.SKIP_PERF_TESTS === 'true' || process.env.CI === 'true';
+
+    const perfDiscoveryMs = process.env.PERF_DISCOVERY_MS
+      ? parseInt(process.env.PERF_DISCOVERY_MS, 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)